sdk/typescript/src/types/objects.ts

```diff
@@ -36,6 +36,21 @@ export type SuiMoveObject = {
   has_public_transfer?: boolean;
 };
 
+export type CoinDenominationInfoResponse = {
+  /** Coin type like "0x2::sui::SUI" */
+  coin_type: string;
+  /** min unit, like MIST */
+  basic_unit: string;
```

Comment:
```suggestion
  basicUnit?: string;
```

Not every coin have a name for basic_unit